common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -147,16 +147,18 @@ object WorkState {
 
   case class Merged(
     sourceIdentifier: SourceIdentifier,
-    numberOfSources: Int
+    modifiedTime: Instant,
+    numberOfSources: Int,
   ) extends WorkState {
 
     type WorkDataState = DataState.Unidentified
-    type TransitionArgs = Int
+    type TransitionArgs = (Option[Instant], Int)
```

Comment:
Under what circumstances would this be `None`?

Reply:
https://github.com/wellcomecollection/catalogue/pull/962#discussion_r509175537

Reply:
"I want to transition the state, and optionally update the modifiedTime when I do that"

Reply:
Ah sorry missed that